scex-core/src/main/scala/com/avsystem/scex/compiler/ScexSettings.scala

```diff
@@ -54,6 +54,9 @@ class ScexSettings extends Settings {
   final val backwardsCompatCacheVersion = StringSetting("-SCEXbackwards-compat-cache-version", "versionString",
     "Additional version string for controlling invalidation of classfile cache", "0")
 
+  final val cacheUnexpectedCompilationExceptions = BooleanSetting("-SCEXcache-unexpected-compilation-exceptions",
+    "Disables the caching of unexpected exceptions thrown during the expression compilation", default = true)
```

Comment:
Needs explanation: what are unexpected exceptions? Is e.g. a syntax error in an expression an unexpected exception?